### PR TITLE
Differentiate between Kubevirt and OpenShift Virtualization

### PR DESCRIPTION
**Pull request by nasark**
Post subclassing OSV, we should differentiate it as a separate provider from Kubevirt in the docs

@miq-bot assign @agrare 
@miq-bot add_labels enhancement, radjabov/yes?
@miq-bot add_reviewers @agrare, @Fryguy

### CODE REVIEW

File: capabilities_matrix/_topics/infrastructure_providers.md [hunk 1]
```diff
@@ -2,67 +2,67 @@
...
-| ------------------------------------------------------------ | ------- | ----------- | ----------------------- | ------------- | -------- |
-| Compute Inventory                                            | ✅      | ✅          | ✅ (Nodes and Services) | ✅            | ✅         |
-| Network Inventory                                            | ✅      | ✅ (OVN)    | ❌                      | ✅            | ❌          |
-| Storage Inventory                                            | ✅      | ❌          | ❌                      | ✅            | ✅         |
-| Events                                                       | ✅      | ✅          | ✅                      | ✅            | ❌          |
-| Metrics                                                      | ✅      | ✅          | ✅                      | ✅            | ❌          |
-| Forensic Analysis (SmartState)                               | ✅      | ✅          | ✅ (Nodes)              | ❌            | ❌          |
+| Discovery                                                    | vSphere | oVirt / RHV | OpenStack undercloud    | IBM Power HMC | KubeVirt | OSV |
```
Comment (agrare) [inline capabilities_matrix/_topics/infrastructure_providers.md:5]: So we show the other subclassed providers together, `oVirt / RHV`
Could we do `| KubeVirt / OSV |` at least until we have any differentiation between the two in features supported

Comment (agrare) [reply capabilities_matrix/_topics/infrastructure_providers.md]: I think we still need to update the capability matrix with KubeVirt / OSV

Comment (nasark) [reply capabilities_matrix/_topics/infrastructure_providers.md]: That is there https://github.com/ManageIQ/manageiq-documentation/blob/master/capabilities_matrix/_topics/infrastructure_providers.md, the earlier change was just to separate it but no longer doing that

Comment (agrare) [reply capabilities_matrix/_topics/infrastructure_providers.md]: :facepalm: I didn't see that in the diff, thanks

**Review by agrare — APPROVED** [review]

**Comment by qahmed1998** [comment]
@Fryguy @agrare 
OpenShift Virtualization is added as a new entry under [capabilities_matrix/_topics/providers.md](https://github.com/ManageIQ/manageiq-documentation/pull/1815/files#diff-4b622ca4a3e3185513cd6030c78a83d7e1aea202ef40b04256667b62d729c408)

So in this table we have two separate entries, one for OpenShift Virtualization and another for KubeVirt.

Keeping the above in mind do we also want to make the changes here as well?
image

i.e The list for Infrastructure Providers becomes:

- Red Hat Virtualization Providers
- OpenStack Infrastructure Providers
- VMware vCenter 
- IBM Power HMC
- OpenShift Virtualization 
- KubeVirt

**Comment by Fryguy** [comment]
Interestingly, Red Hat Virtualization and oVirt are in a similar case where they are really basically the same. In other places we just put them together to avoid the redundancy, (e.g. RHV/oVirt), so I personally prefer that for OSV/KubeVirt

**Comment by qahmed1998** [comment]
Thanks Jason. I am adding my approval for the PR in that case.

**Review by qahmed1998 — APPROVED** [review]

**Comment by Fryguy** [comment]
Backported to `radjabov` in commit 6a1c53bd27827a3689362c23eee015d02e5f3de4.

```text
commit 6a1c53bd27827a3689362c23eee015d02e5f3de4
Author: Adam Grare <adam@grare.com>
Date:   Mon Oct 28 08:32:46 2024 -0400

    Merge pull request #1815 from nasark/osv_provider_changes
    
    Differentiate between Kubevirt and OpenShift Virtualization
    
    (cherry picked from commit df50a8cf4c98236edd8cfdd4859f42e731dda5fb)
```

**Pull request merged**